File: Problems 1-9/p6.py
import time


def main():
    t1 = time.perf_counter()
    r = abs(sum(range(1, 101)) ** 2 - sum(map(lambda x: x**2, range(1, 101))))
    t2 = time.perf_counter()
    return r, t2 - t1

File: Problems 1-9/p6.jl
"""
Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
"""

main() = abs(sum(1:100)^2 - (sum(x -> x^2, 1:100)))

"""
julia> @btime main()
  3.291 ns (0 allocations: 0 bytes)
25164150
"""